zebrad/src/application.rs

```diff
@@ -235,10 +235,12 @@ impl Application for ZebradApp {
         let mut components = self.framework_components(command)?;
 
         // Load config *after* framework components so that we can
-        // report an error to the terminal if it occurs.
+        // report an error to the terminal if it occurs (unless used with the 'generate' cmd).
```

Comment:
Nitpick, I'll just merge and approve.

```suggestion
        // report an error to the terminal if it occurs (unless used with a command that doesn't need the config).
```